interop/tapir/src/main/scala/caliban/interop/tapir/TapirAdapter.scala

```diff
@@ -190,4 +196,13 @@ object TapirAdapter {
 
   def isFtv1Header(r: Header): Boolean =
     r.name == GraphQLRequest.`apollo-federation-include-trace` && r.value == GraphQLRequest.ftv1
+
+  private def computeCacheDirective(extensions: Option[ResponseValue.ObjectValue]): Option[String] =
```

Comment:
Shall we keep it in extensions after we used it?

Reply:
Yeah probably not necessary, I'll strip it from the extensions.